src/app/services/tasks-by-status.service.ts

```diff
@@ -13,11 +13,19 @@ export class TasksByStatusService {
   #storageService = inject(StorageService);
 
   readonly defaultStatuses: TaskStatusColored[] = this.#defaultConfigService.defaultTasksByStatus;
-
+  /**
+   * 
+   * @param table 
+   * @returns An array of TaskStatusColored objects
+   */
   restoreStatuses(table: TableTasksByStatus): TaskStatusColored[] {
```

Comment:
I don't think this documentation is necessary, since it doesn't provide anything the dev doesn't already know.
```suggestion
  restoreStatuses(table: TableTasksByStatus): TaskStatusColored[] {
```